specs/executionAPISpec.md

```diff
@@ -0,0 +1,530 @@
+* <a href="#feeHistory">eth_feeHistory</a>
+* <a href="#blockNumber">eth_blockNumber</a>
+* <a href="#sign">eth_sign</a>
+* <a href="#call">eth_call</a>
+* <a href="#fillTransaction">eth_fillTransaction</a>
+* <a href="#createAccessList">eth_createAccessList</a>
+* <a href="#estimateGas">eth_estimateGas</a>
+* <a href="#signTransaction">eth_signTransaction</a>
+* <a href="#sendRawTransaction">eth_sendRawTransaction</a>
+* <a href="#sendTransaction">eth_sendTransaction</a>
+* <a href="#pendingTransactions">eth_pendingTransactions</a>
+* <a href="#resend">eth_resend</a>
+* <a href="#getProof">eth_getProof</a>
+* <a href="#getBalance">eth_getBalance</a>
+* <a href="#getTransactionCount">eth_getTransactionCount</a>
+* <a href="#getStorageAt">eth_getStorageAt</a>
+* <a href="#getCode">eth_getCode</a>
+* <a href="#getBlockByHash">eth_getBlockByHash</a>
+* <a href="#getBlockByNumber">eth_getBlockByNumber</a>
+* <a href="#getHeaderByNumber">eth_getHeaderByNumber</a>
+* <a href="#getHeaderByHash">eth_getHeaderByHash</a>
+* <a href="#getBlockTransactionCountByHash">eth_getBlockTransactionCountByHash</a>
+* <a href="#getBlockTransactionCountByNumber">eth_getBlockTransactionCountByNumber</a>
+* <a href="#getTransactionReceipt">eth_getTransactionReceipt</a>
+* <a href="#getTransactionByHash">eth_getTransactionByHash</a>
+* <a href="#getTransactionByBlockHashAndIndex">eth_getTransactionByBlockHashAndIndex</a>
+* <a href="#getTransactionByBlockNumberAndIndex">eth_getTransactionByBlockNumberAndIndex</a>
+* <a href="#getRawTransactionByHash">eth_getRawTransactionByHash</a>
+* <a href="#getRawTransactionByBlockHashAndIndex">eth_getRawTransactionByBlockHashAndIndex</a>
+* <a href="#getRawTransactionByBlockNumberAndIndex">eth_getRawTransactionByBlockNumberAndIndex</a>
+* <a href="#getUncleCountByHash">eth_getUncleCountByHash</a> 
+* <a href="#getUncleCountByNumber">eth_getUncleCountByNumber</a>
+* <a href="#getUncleByBlockHashAndIndex">eth_getUncleByBlockHashAndIndex</a> 
+* <a href="#getUncleByBlockNumberAndIndex">eth_getUncleByBlockNumberAndIndex</a>
+* <a href="#newFilter">eth_newFilter</a>
+* <a href="#newBlockFilter">eth_newBlockFilter</a>
+* <a href="#newPendingTransactionFilter">eth_newPendingTransactionFilter</a>
+* <a href="#uninstallFilter">eth_uninstallFilter</a>
+* <a href="#getFilterChanges">eth_getFilterChanges</a>
+* <a href="#getFilterLogs">eth_getFilterLogs</a>
+* <a href="#getLogs">eth_getLogs</a>
+___
+## <p id="clientVersion">web3_clientVersion</p>
+* [WC-1] web3_clientVersion **MUST** return a string containing information about the client version.
+* [WC-2] web3_clientVersion **MUST** start with the name of the client.
+* [WC-3] web3_clientVersion **MUST** return the client version after the name.
+* [WC-4] web3_clientVersion **MUST** return the operating system and architecture the client is running on.
+* [WC-5] web3_clientVersion **MUST** return the name of the language being used.
```

Comment:
```suggestion
* **MUST** return a string containing information about the client version.
* **MUST** start with the name of the client.
* **MUST** return the client version after the name.
* **MUST** return the operating system and architecture the client is running on.
* **MUST** return the name of the language being used.
```
I think the prefix and repeating the method name is unnecessary and makes reading harder in this context.  We already know we are talking about `web3_clientVersion` because that is the section we are in, and the `[WC-#]` doesn't improve readability I don't think.  Maybe the `[WC-#]` is used by the renderer in some way?  If so, then I recommend avoiding abbreviations and using full words so at least we don't have to worry about name collisions.

Reply:
Same comment for all other sections.